pkgs/applications/networking/irc/weechat/scripts/colorize_nicks/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchurl }:
```

lib can be replaced with stdenv.lib.

---

I don't feel like indirections like this are the way to go.

---

```suggestion
{ pkgs }:
```
Then we can use `pkgs.stdenv`, `pkgs.lib`, `pkgs.fetchurl`?
`</kidding>`

More to the point. Do we have any guidelines as to what to prefer here? I think I prefer the solution present in the PR.